client/state/signup/steps/website-content/reducer.ts

```diff
@@ -13,234 +13,224 @@ import {
 	SIGNUP_STEPS_WEBSITE_CONTENT_REMOVE_LOGO_URL,
 	SIGNUP_STEPS_WEBSITE_CONTENT_FEEDBACK_CHANGE,
 } from 'calypso/state/action-types';
-import { withSchemaValidation } from 'calypso/state/utils';
 import { getSingleMediaPlaceholder, MediaUploadedData } from './actions';
-import {
-	schema,
-	initialState,
-	WebsiteContentCollection,
-	MEDIA_UPLOAD_STATES,
-	LOGO_SECTION_ID,
-} from './schema';
+import { initialState, LOGO_SECTION_ID, MEDIA_UPLOAD_STATES } from './constants';
+import { WebsiteContentCollectionState } from './types';
 import type { AnyAction } from 'redux';
```

Comment:
Github shows this is as a large diff, but it is majorly an indentation change due to the removal of `withSchemaValidation`.

The code for only `SIGNUP_STEPS_WEBSITE_CONTENT_INITIALIZE_PAGES` action type has been changed. This the redux state is no longer the source of truth, I have removed the `siteId` checks that caused issues with persisted state data before.